exercises/practice/rational-numbers/.meta/Example.roc

```diff
@@ -0,0 +1,55 @@
+module [add, sub, mul, div, abs, exp, expReal, reduce]
+
+add : [Rational (Int a) (Int a)], [Rational (Int a) (Int a)] -> [Rational (Int a) (Int a)]
```

Comment:
Are the square brackets necessary here?

Reply:
Mmmh, that's interesting, I put them here because without them I was getting an error message in VSCode (presumably coming from the language server):
```
UNRECOGNIZED NAME

Nothing is named `Rational` in this scope.

3│  add : Rational (Int a) (Int a), [Rational (Int a) (Int a)] -> [Rational (Int a) (Int a)]
          ^^^^^^^^^^^^^^^^^^^^^^^^

Did you mean one of these?

    Decimal
    expReal
    Bool
    Integer
```

I didn't try running the code as I assumed there was an error. But I just tried, and it actually works fine. So I'll remove the brackets, but I think there might be a bug in the LSP.

Reply:
Does `()` instead of `[]` work?

Reply:
Maybe that won't work either, because the underline looks correct

Reply:
Yeah looking at the repl:
```roc
» Red

Red : [Red]
```
The original seems correct :+1: